content/news/2023-06-26-elixir-ahm/index.md

```diff
@@ -0,0 +1,79 @@
+---
+title: 'Freiburg Galaxy team at the ELIXIR All Hands Meeting 2023'
+date: "2023-06-16"
+tease: "From June 5th to 8th, several members from the Freiburg Galaxy team travelled to Dublin, Ireland to participant and present their work at the ELIXIR All Hands Meeting 2023"
+hide_tease: false
+subsites: [all-eu]
+main_subsite: eu
+---
+
+During 5-8th of June, members of the European Galaxy team travelled to Dublin to join the ELIXIR All Hands meeting 2023. Some of us travelled by flight, but Björn and Wolgang took the ferry: 
+
+<twitter tweet="1664969165667463174"></twitter>
+
+In Dublin, we joined other colleagues from ELIXIR-DE
+
+<twitter tweet="1666766756738736131"></twitter>
+
+All over the 4 days of the meeting, there was a **broad presence of Galaxy** across the different workshops, symposiums, plenary sessions, and posters. We will highlight in this post some of this presence.
+
+# Talks
+
+## Day 1 - June 5
+
+On the first day, Wolfgang gave the talk *"Galaxy - Galaxy workflows for taking viral genomic surveillance beyond SARS-CoV-2"*, in the [*Biodiversity, Food Security and Pathogens*](https://docs.google.com/document/d/1GOGaZdlLUD7gbg5HFlYWOmv4Nfppk1L0y2b3w1s-w1U/edit#heading=h.dqg3p1q9y270) mini-symposium.
```

Comment:
```suggestion
On the first day, Wolfgang gave the talk [*Galaxy - Galaxy workflows for taking viral genomic surveillance beyond SARS-CoV-2*](https://docs.google.com/presentation/d/1l7eGLXa3Rah4uBHZdf2wtUs3jMMk0eox9CK2lb48Ck4/), in the [*Biodiversity, Food Security and Pathogens*](https://docs.google.com/document/d/1GOGaZdlLUD7gbg5HFlYWOmv4Nfppk1L0y2b3w1s-w1U/) mini-symposium.
```

Like the link to Pavan's presentation, this actually takes you to the full slide deck of the event, not just my slides, but probably ok like this?